src/app/dashboard/dashboard.component.html

```diff
@@ -0,0 +1,54 @@
+<div id="greeting"><b>Hi, {{name}}</b></div>
+<br>
+<div class="row">
+    <div class="col-md-6 table-responsive">
+        <h3 i18n>Community List</h3>
+        <table class="table">
+            <thead>
+            <tr>
+                <th i18n>Community</th>
+                <th i18n>Version No.</th>
+                <th i18n>Nation Name</th>
+                <th i18n>Status</th>
+            </tr>
+            </thead>
+            <tbody>
+                <tr *ngFor="let community of communities">
+                    <td>
+                        {{community.doc.name}}
+                    </td>
+                    <td>
+                        {{community.doc.version}}
+                    </td>
+                    <td>
+                        {{community.doc.nationName}}
+                    </td>
+                    <td i18n>
+                        {{community.doc.registrationRequest}}
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+        <button i18n class="btn btn-primary" type="viewAll" (click)="viewCommunity()">View All</button>
+    </div>
+    <div class="col-md-6 table-responsive">
+        <h3 i18n>Nation List</h3>
+        <table class="table ">
+            <thead>
+            <tr>
+                <th i18n>Nation Name</th>
+                <th i18n>Admin Name</th>
+                <th i18n>Nation Url</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr *ngFor="let nations of nation; let i = index">
+                <td>{{ nations.doc.name }}</td>
```

Comment:
The name for nations should be `nation_name`.  Mock data is fixed to use `nation_name` in #154

Reply:
@paul nation name has been changed to nation_name